Updates for the security section (#168)
neo4j/docs-graphql: modules/ROOT/pages/security/authorization.adoc

}
----

> [NOTE]

Reviewer: ```suggestion
[NOTE]
====
```

> If there is no `operations` argument with a list of operations, the GraphQL Library treats the authorization configuration as if the full list of operations had been provided.

Reviewer: ```suggestion
In case there is no `operations` argument with a list of operations, the GraphQL Library treats the authentication configuration as if the full list of operations had been provided.
====
```

Author: it's authorization here, at least i think it is

Reviewer 2: Yes this page is for the `authorization` directive.

Reviewer 2: Yes this page is for the `authorization` directive.

> If there is no `operations` argument with a list of operations, the GraphQL Library treats the authorization configuration as if the full list of operations had been provided.

Author: ```suggestion
In case there is no `operations` argument with a list of operations, the GraphQL Library treats the authorization configuration as if the full list of operations had been provided.
====
```

}
----

[NOTE]

> If there is no `operations` argument with a list of operations, the GraphQL Library treats the authorization configuration as if the full list of operations had been provided.

Reviewer: ```suggestion
====
In case there is no `operations` argument with a list of operations, the GraphQL Library treats the authentication configuration as if the full list of operations had been provided.
====
```

Author: same here, i think it's authorization

> If there is no `operations` argument with a list of operations, the GraphQL Library treats the authorization configuration as if the full list of operations had been provided.

Author: ```suggestion
====
In case there is no `operations` argument with a list of operations, the GraphQL Library treats the authorization configuration as if the full list of operations had been provided.
====
```
